quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -583,6 +613,28 @@ impl Ingester {
                 persist_successes.push(persist_success);
             }
         }
+        if !shards_to_close.is_empty() {
+            for queue_id in &shards_to_close {
+                let shard = state_guard
+                    .shards
+                    .get_mut(queue_id)
+                    .expect("shard should exist");
+
+                shard.shard_state = ShardState::Closed;
```

Comment:
when/through which path change of state is this reported to the control plane -> routers?